config/default_values.yml

```diff
@@ -0,0 +1,12 @@
+bank_accounts:
+  - Current
+categories:
+  1:
```

Comment:
@dissident http://stackoverflow.com/questions/17173864/how-to-make-a-list-of-associative-array-in-yaml